### PR TITLE
Feature: List entries

### PR DESCRIPTION
**Pull request by paulotijero**
solved #2

### CODE REVIEW

File: controller/list_methods.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative 'entry_methods' 
+require "date"
+
+def list_daily
+  list = []
+  data = read_data
+  data.each do |val|
+    list.push(val[1])
+  end
+  list.sort_by{|value| value["datetime"].to_i}
```
Comment (deyviconde) [inline controller/list_methods.rb]: Great, this order our post by datetime. Extremely helpful !!

File: public/css/style.css [hunk 2]
```diff
@@ -24,14 +24,13 @@ body {
 .card-list {
   background-color: #eaeaea;
   border-radius: 5px;
-  margin: 3%;
-  padding: 5%;
+  margin: 10px 5px;
+  padding: 5px;
```
Comment (deyviconde) [inline public/css/style.css]: Why did you change 5% to 5px. I have same question for  margin

File: views/list.erb [hunk 3]
```diff
@@ -2,111 +2,17 @@
   <div class="card-block card-shadow">
     <div class="row">
       <div class="col-sm-6">
-        <p class="title-option">List</p>
+        <a class="menu" href="/">
+          <p class="title-option <%= "select" if content_for?(:list_entry) %>">Daily</p>
```
Comment (deyviconde) [inline views/list.erb]: Awesome, I like conditionals in one line

File: views/list.erb [hunk 4]
```diff
@@ -2,111 +2,17 @@
...
-                  aria-hidden="true"
-                ></span>
-          </div>
-        </div>
+        <a class="menu" href="/photo">
+          <p class="title-option <%= "select" if content_for?(:add_photo) %>">Photo</p>
+        </a>
       </div>
     </div>
+    <%= yield_content :add_photo if content_for?(:add_photo) %>
```
Comment (deyviconde) [inline views/list.erb]: For two cases works great, but how about if we have 100 ?

File: route.rb [hunk 5]
```diff
@@ -2,11 +2,37 @@
 require "sinatra/reloader"
 require "sinatra/content_for"
 require "erb"
+require "./controller/list_methods.rb"
 
-set :port,8000
+set :port,8001
```
Comment (liannivin) [inline route.rb]: This change was not necessary, but when I pull this change I will fix that.

File: route.rb [hunk 6]
```diff
@@ -2,11 +2,37 @@
...
 
 get "/" do
-  erb :list, {:layout=> :layout}
-  erb :daily, {:layout=> :layout}
```
Comment (liannivin) [inline route.rb]: <% yield %> makes this unnecessary.

File: data/entries.json [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    "title": "Titulo 4 de prueba",
+    "datetime": "20190320220428",
+    "content": "Es un hecho establecido hace demasiado tiempo que un lector se distraerá con el contenido del texto de un sitio mientras que mira su diseño. El punto de usar Lorem Ipsum es que tiene una distribución más o menos normal de las letras, al contrario de usar textos como por ejemplo \"Contenido aquí, contenido aquí\". Estos textos hacen parecerlo un español que se puede leer. Muchos paquetes de autoedición y editores de páginas web usan el Lorem Ipsum como su texto por defecto, y al hacer una búsqueda de \"Lorem Ipsum\" va a dar por resultado muchos sitios web que usan este texto si se encuentran en estado de desarrollo. Muchas versiones han evolucionado a través de los años, algunas veces por accidente, otras veces a propósito (por ejemplo insertándole humor y cosas por el estilo).",
+    "content_before": [],
+    "images": [],
+    "highlight": 0,
+    "is_deleted": 1,
+    "deleted_datetime": "20190320230420"
+  }
+}
```
Comment (MariethPerez) [inline data/entries.json]: @paulotijero the test data shouldn't have been uploaded, but I will download and clear data.

File: public/css/style.css (hunk 2 above)
Comment (paulotijero) [reply public/css/style.css]: To use less space, consider using screen pixels instead of a percentage

File: views/list.erb (hunk 4 above)
Comment (paulotijero) [reply views/list.erb]: Good question!!, I think we could duplicate this line, maybe in the future we will find a better solution :nerd_face:

File: route.rb (hunk 5 above)
Comment (paulotijero) [reply route.rb]: It was necessary for me to run in my local

File: route.rb (hunk 6 above)
Comment (paulotijero) [reply route.rb]: if we have a view called "layout", it is not necessary to add it since sinatra recognizes it and automatically calls it

File: data/entries.json (hunk 7 above)
Comment (paulotijero) [reply data/entries.json]: It's okay

File: controller/list_methods.rb (hunk 1 above)
Comment (KattyaCuevas) [inline controller/list_methods.rb]: This method `sort_by` is not saving the ordered list.
